pom.xml

```diff
@@ -1471,7 +1471,7 @@
     <java.min.version>${compileSource}</java.min.version>
     <!-- Dependencies -->
     <hadoop-two.version>2.8.5</hadoop-two.version>
-    <hadoop-three.version>3.1.2</hadoop-three.version>
+    <hadoop-three.version>3.1.4-SNAPSHOT</hadoop-three.version>
```

Comment:
no depending on snapshot versions in the committed codebase. we don't include references to snapshot repositories, so we shouldn't be able to find any.